### PR TITLE
Dev/howtostart

### PR DESCRIPTION
**Pull request by McCambley**
No loom this time! 🎉

Here's the "How to Start" section - One section, two new components 
- The someone tricky alignment of the steps and corresponding numbers was handled with `grid-template-area`, which I'm beginning to love. 
- The aspect ratio of the images changes at the mobile viewpoint. If you look at the `HowToCard.css` style sheet you'll see some weird scaling. It doesn't seem ideal, but it appeared to look good at all breakpoints. 
- The component's button is connected to the form as well 👍

_**I did make changes to the header in this PR!**_
- I specified what properties are to be transitioned when using the `transition` property in order to avoid some unintended transitions. There's a video of that in our Slack channel. I lied... one loom. 

Other than that, pretty straight forward! 

https://user-images.githubusercontent.com/74033573/130310618-b49186bf-f28f-4f13-b022-8b181951b951.mov

Thanks y'all! ✌️

### CODE REVIEW

File: src/components/how-to-card/HowToCard.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import './HowToCard.css';
+
+export default function HowToCard({ step, title, subtitle, image, alt }) {
```
Comment (denisehung) [inline src/components/how-to-card/HowToCard.js:6]: Looks good! For the Requests component I didn't create a separate file for the cards though (cause it's bascially just a heading and a container with 3 cards). Do you think I should do that too?

File: src/components/how-to-start/HowToStart.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            key={card.id}
+            step={index + 1}
+            title={card.title}
+            subtitle={card.subTitle}
+            image={card.image}
+            alt={card.alt}
+          />
+        ))}
+      </div>
+      <button type="button" className="how__button" onClick={handleButtonClick}>
```
Comment (denisehung) [inline src/components/how-to-start/HowToStart.js:30]: This button is missing a hover animation. I applied the following properties to the CTA-button in the Hero and Form component to make the button enlarge on hover. Could you apply these too?

.how__button {
  transition: all 0.3s ease;
}

.how__button:hover {
  transform: translate3D(0, -1px, 0) scale(1.05);
}

File: src/components/how-to-start/HowToStart.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  line-height: 56px;
+  text-align: center;
+  font-weight: 400;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.how__content {
+  display: grid;
+  grid-template-areas:
```
Comment (denisehung) [inline src/components/how-to-start/HowToStart.css:23]: Nice how you did this with grid template areas! 🎉

**Comment by denisehung** [comment]
Hi Jake, great job! I only added a few comments; adding the hover animation to the CTA-button is most important. Think we can merge this component once you've fixed that!

**Review by denisehung — CHANGES_REQUESTED** [review]
> This button is missing a hover animation. I applied the following properties to the CTA-button in the Hero and Form component to make the button enlarge on hover. Could you apply these too?

.how__button {
  transition: all 0.3s ease;
}

.how__button:hover {
  transform: translate3D(0, -1px, 0) scale(1.05);
}

**Review by thinklikeadesigner — CHANGES_REQUESTED** [review]

File: src/components/how-to-card/HowToCard.js (hunk 1 above)
Comment (McCambley) [reply src/components/how-to-card/HowToCard.js:6]: Nah I think it's ok. If you want to go for it but it's not a breaking issue!

File: src/components/how-to-start/HowToStart.js (hunk 2 above)
Comment (McCambley) [reply src/components/how-to-start/HowToStart.js:30]: I'll fix this in the button component branch!

File: src/components/how-to-start/HowToStart.css (hunk 3 above)
Comment (McCambley) [reply src/components/how-to-start/HowToStart.css:23]: Thanks!!!

File: src/components/how-to-start/HowToStart.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  max-width: 1440px;
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+.how__title {
```
[the remark this answers is not in the document]
Comment (McCambley) [reply src/components/how-to-start/HowToStart.css:12]: Truuuuueee ok I'll look into it, good catch.

**Comment by McCambley** [comment]
I added a few crazy breakpoints and I think it looks a tad better. Crazy code... maybe not so pretty... but it works?

**Pull request merged**